cmake/BundledAwsSdkCpp.cmake

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 function(use_bundled_libaws SOURCE_DIR BINARY_DIR)
-    set(PATCH_FILE "${SOURCE_DIR}/thirdparty/aws-sdk-cpp/core-wstr-fix.patch")
+    set(PATCH_FILE "${SOURCE_DIR}/thirdparty/aws-sdk-cpp/aws-sdk-cpp.patch")
```

Comment:
it would be better IMO, of the name of the patch file indicated its purpose (that it solves c++20 compilation problems)

Reply:
Yes, good idea.  I changed it to `c++20-compilation-fixes.patch` in https://github.com/apache/nifi-minifi-cpp/pull/1150/commits/9c90a834d446c9f636a8234748518accf6d48a89.